torch/testing/_internal/common_distributed.py

```diff
@@ -1502,24 +1500,29 @@ def _run(
 
 class MultiProcContinousTest(TestCase):
     # Class variables:
+    MAIN_PROCESS_RANK = -1
     # number of test processes
-    world_size: int = 2
+    world_size: int = -2  # unset state
     # rank of the current process
-    rank: int = -1  # unset state
+    rank: int = -2  # unset state
     # Rendezvous file
     rdvz_file: Optional[str] = None
     # timeout configured per class
     timeout: timedelta = timedelta(seconds=120)
 
     @classmethod
-    @abc.abstractmethod
-    def backend_str(cls) -> str:
+    def backend_str(cls) -> str | None:
```

Comment:
PyTorch is still using Python 3.9, which doesn't support `|` (this supports in Python 3.10)